### ISSUE TITLE
[Bug Report] Icon inside not visible when v-text-field is solo-inverted

### ISSUE THREAD

**Issue by icebeam030** [vuetifyjs/vuetify#8827]
### Environment
**Vuetify Version:** 2.0.11
**Vue Version:** 2.6.10
**Browsers:** Chrome 74.0.3729.169
**OS:** Windows 10

### Steps to reproduce
See Codepen. Just add a v-toolbar and inside add a clearable and solo-inverted v-text-field.

### Expected Behavior
After typing in some text, the clear button should be visible at the right end.

### Actual Behavior
Cannot see the clear icon once the text field is activated, although it can still be clicked. The icon is visible though when the text field is deactivated, or is not solo-inverted.

### Reproduction Link
<a href="https://codepen.io/icebeam030/pen/gOYGpVG" target="_blank">https://codepen.io/icebeam030/pen/gOYGpVG</a>

**Comment by nickbg321** [comment]
This issue is also reproducible on the [Google Contacts](https://vuetifyjs.com/examples/layouts/google-contacts) pre-made layout. 

It does not seem to be related to the clearable functionality, but more so with solo-inverted.

The field just turns white, making everything else inside it invisible.

**Comment by icebeam030** [comment]
> This issue is also reproducible on the [Google Contacts](https://vuetifyjs.com/examples/layouts/google-contacts) pre-made layout.
> 
> It does not seem to be related to the clearable functionality, but more so with solo-inverted.
> 
> The field just turns white, making everything else inside it invisible.

Agreed. Edited accordingly.

**Comment by ThoGro** [comment]
You can use the `color` property. This sets the inner-icon color for the focused text field.

For Example:

```
<v-text-field
          class="mx-6"
          placeholder="Search"
          flat
          clearable
          hide-details
          solo-inverted
          color="pink"
></v-text-field>
```

**Comment by johnleider** [comment]
Inverted solo fields are not in a good place right now but I would like to get a few QOL fixes in starting with #9534
